HW_2/templates/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Введите имя и email</title>
+</head>
+<body>
+    <h2>Введите имя и email</h2>
+    <form method="post" action="{{ url_for('welcome') }}">
```

Comment:
После этого тут можно изменить
```suggestion
    <form method="post" action="{{ url_for('index') }}">
```
либо вообще убрать атрибут `action`